recipes/pm4py/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "pm4py" %}
+{% set version = "2.2.19.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/ab/19/aa8e3014e9d606a9cf81dc435c4fd39aa89a3f57c6d64c55c730c36685b3/pm4py-2.2.19.1.tar.gz
+  sha256: 2104e253e16720706896df2cb359a85fef09d219da923b0a093ab307d4dc4d17
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy
+    - pandas
+    - deprecation
+    - lxml
+    - scipy
+    - graphviz
+    - intervaltree
+    - matplotlib-base
+    - networkx
+    - tqdm
+    - cvxopt
+  run:
+    - python
+    - numpy
+    - pandas
+    - deprecation
+    - lxml
+    - scipy
+    - graphviz
+    - intervaltree
+    - matplotlib-base
+    - networkx
+    - tqdm
+    - cvxopt
+
+test:
+  imports:
+    - pm4py
+
+about:
+  home: https://github.com/pm4py/pm4py-core
```

Comment:
Is there another derivative of this? Why is it called pm4py-core in the source but not here?